src/utility/SuggestionsManager.ts

```diff
@@ -1,11 +1,13 @@
 import {ActBotConstants} from "../Constants";
 import fs = require("fs");
+import { Collection } from "discord.js";
+import { SuggestionLists} from "./SuggestionLists";
 
 /**
- * Defines the delimiter to separate suggestions in file
+ * Enum for potential results
  * @constant
  */
-const delimiter: string = "~\n";
+const resultEnum = {LISTNOTEMPTY: 9, LISTREMOVED: 8, SUGGESTIONREMOVED: 7, LISTEXISTS: 6, SUGGESTIONDOESNOTEXIST: 5, LISTADDED: 4, DERP: 3, SUGGESTIONADDED: 2, LISTDOESNOTEXIST: 1, SUGGESTIONEXISTS: 0};
```

Comment:
Definitely make this an enum type and have it the function using this return the type rather than a "number"